openfisca_uk/variables/hmrc/income_tax/allowances/marriage_allowance.py

```diff
@@ -0,0 +1,69 @@
+from openfisca_uk.model_api import *
+
+
+class unused_personal_allowance(Variable):
+    value_type = float
+    entity = Person
+    label = u"Unused personal allowance"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        return max_(
+            person("personal_allowance", period)
+            - person("adjusted_net_income", period),
+            0,
+        )
+
+
+class meets_marriage_allowance_income_conditions(Variable):
+    label = "Meets Marriage Allowance income conditions"
+    documentation = "Whether this person (and their partner) meets the conditions for this person to be eligible for the Marriage Allowance, as set out in the Income Tax Act 2007 sections 55B and 55C"
+    entity = Person
+    definition_period = YEAR
+    value_type = bool
+    reference = "https://www.legislation.gov.uk/ukpga/2007/3/section/55B"
+
+    def formula(person, period):
+        tax_band = person("tax_band", period)
+        bands = tax_band.possible_values
+        band = person("tax_band", period)
```

Comment:
```suggestion
        return ~np.isin(tax_band, [bands.HIGHER, bands.ADDITIONAL])
```

Reply:
Oh sorry, forgot this doesn't seem to work (Enums in OpenFisca? Will check out later)